### PR TITLE
Changed function signature from rest.API to rest.Send

### PR DESCRIPTION
**Pull request by adnrs96**
This PR changes the function signature from API to Send as raised in issue #13 .
This must be able to resolve issue #13 .

### CODE REVIEW

File: rest.go [hunk 1]
```diff
@@ -90,8 +90,8 @@ func BuildResponse(res *http.Response) (*Response, error) {
 }
 
 // API is the main interface to the API.
```
Comment (thinkingserious) [inline rest.go]: Please update this comment.

File: rest.go [hunk 2]
```diff
@@ -103,7 +103,7 @@ func (c *Client) MakeRequest(req *http.Request) (*http.Response, error) {
...
```
Comment (thinkingserious) [inline rest.go]: Please update this comment

**Comment by thinkingserious** [comment]
Hi @adnrs96,

This looks great now, thanks!

I will be merging this after we get [this one](https://github.com/sendgrid/sendgrid-go/issues/94) closed.

**Comment by thinkingserious** [comment]
Hello @adnrs96,

I think that in order to avoid a breaking change, we should leave `API` as a pass through function to `Send`. Do you mind making that change?

With Best Regards,

Elmer

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/rest?pullRequest=14) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/rest?pullRequest=14) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: adnrs96<br/>:x: thinkingserious<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/rest?pullRequest=14) it.</sub>

**Pull request closed without merging**